docs/core_docs/docs/modules/model_io/output_parsers/types/openai_tools.ipynb

```diff
@@ -160,7 +160,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import { JsonOutputToolsParser } from \"langchain/output_parsers\";\n",
+    "import { JsonOutputToolsParser } from \"@langchain/core/output_parsers/openai_tools\";\n",
```

Comment:
Can we just import from `@langchain/core/output_parsers` instead for simplicity?

Reply:
ah yeah this would work too, not sure if we need another release b/c I don't think that entrypoint exists yet. Will check